crates/evm/src/evm/executor.rs

```diff
@@ -37,6 +39,22 @@ where
         *self.evm.tx_mut() = create_tx_env(tx);
         self.evm.transact_commit()
     }
```

Comment:
dont need this now?

Reply:
we are still using that when executing system transactions